### PR TITLE
Minor. Update api doc about source newly updatable fields. RD-23635

### PR DESCRIPTION
**Pull request by onima**
This will be displayed only on 1 endpoint for source -> PUT  content_sources/{sourceId}

### CODE REVIEW

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -976,6 +1000,180 @@ paths:
...
+              - CLIENT_ONCE
+              - FAIL_IF_NO_PEER_CERT
+            type: string
+        - description: Used to identify HELO/EHLO when connecting to the SMTP server. Only on Email sources
+          in: query
+          name: smtp_helo_domain
+          required: false
+          schema:
+            type: string
+        - description: Threading_heuristics. Only on Email sources
```
Comment (kalilz4485) [inline specs/engage-digital_openapi3.yaml:1147]:
```suggestion
        - description: Threading heuristics. Only on Email sources

```

File: specs/engage-digital_openapi3.yaml [hunk 2]
```diff
@@ -976,6 +1000,180 @@ paths:
...
+          required: false
+          schema:
+            type: integer
+        - description: IMAP username. Only on Email sources
+          in: query
+          name: imap_username
+          required: false
+          schema:
+            type: string
+        - description: IMAP password. Provide associated IMAP authentication type -> `basic`. Only on Email sources
```
Comment (kalilz4485) [inline specs/engage-digital_openapi3.yaml:1035]:
```suggestion
        - description: IMAP password. Used when IMAP authentication type is `basic`. Only on Email sources
```

File: specs/engage-digital_openapi3.yaml [hunk 3]
```diff
@@ -976,6 +1000,180 @@ paths:
...
+          in: query
+          name: imap_password
+          required: false
+          schema:
+            type: string
+        - description: IMAP OAuth 2.0 refresh token. Provide associated IMAP authentication type -> `microsoft_identity_platform`. Only on Email sources
```
Comment (kalilz4485) [inline specs/engage-digital_openapi3.yaml:1041]:
```suggestion
        - description: IMAP OAuth 2.0 refresh token. Used when IMAP authentication type is `microsoft_identity_platform`. Only on Email sources

```

File: specs/engage-digital_openapi3.yaml [hunk 4]
```diff
@@ -976,6 +1000,180 @@ paths:
...
+          required: false
+          schema:
+            type: boolean
+        - description: Use STARTTLS protocol to establish secure IMAP connection. Only on Email sources
+          in: query
+          name: imap_use_start_tls
+          required: false
+          schema:
+            type: boolean
+        - description: 'IMAP Authentication type ("basic" or "microsoft_identity_platform"). Provide either associated imap_password or imap_oauth_refresh_token. Only on Email sources'
```
Comment (kalilz4485) [inline specs/engage-digital_openapi3.yaml:1059]:
```suggestion
        - description: 'IMAP Authentication type ("basic" or "microsoft_identity_platform"). Only on Email sources'

```

File: specs/engage-digital_openapi3.yaml [hunk 5]
```diff
@@ -976,6 +1000,180 @@ paths:
...
+          required: false
+          schema:
+            type: integer
+        - description: SMTP username. Only on Email sources
+          in: query
+          name: smtp_username
+          required: false
+          schema:
+            type: string
+        - description: SMTP password. Provide associated SMTP authentication type -> `basic`. Only on Email sources
```
Comment (kalilz4485) [inline specs/engage-digital_openapi3.yaml:1097]:
```suggestion
        - description: SMTP password. Used when SMTP authentication type is `basic`. Only on Email sources

```

File: specs/engage-digital_openapi3.yaml [hunk 6]
```diff
@@ -976,6 +1000,180 @@ paths:
...
+          in: query
+          name: smtp_password
+          required: false
+          schema:
+            type: string
+        - description: SMTP OAuth 2.0 refresh token. Provide associated SMTP authentication type -> `microsoft_identity_platform`. Only on Email sources
```
Comment (kalilz4485) [inline specs/engage-digital_openapi3.yaml:1103]:
```suggestion
        - description: SMTP OAuth 2.0 refresh token. Used when SMTP authentication type is `microsoft_identity_platform`. Only on Email sources

```

File: specs/engage-digital_openapi3.yaml [hunk 7]
```diff
@@ -976,6 +1000,180 @@ paths:
...
+          in: query
+          name: spam_assassin_level
+          required: false
+          schema:
+            enum:
+              - disabled
+              - relaxed
+              - strict
+            type: string
+        - description: Whitelisted emails. All emails received to this address won't be threaded and can be associated with an anonymous identity or the reply_to, if there is one. Only on Email sources
```
Comment (kalilz4485) [inline specs/engage-digital_openapi3.yaml:1163]:
```suggestion
        - description: Whitelisted emails. All emails received from this address won't be threaded and can be associated with an anonymous identity or the Reply-To, if there is one. To be filled when implementing a form. Only on Email sources

```

**Comment by kalilz4485** [comment]
LGTM

**Comment by onima** [comment]
I have just updated documentation to take into account newly added fields -> emall_parser, default_task_priority, imap_mailbox_used_for_sent_mail, time_for_thread_inclusion_enabled and time_for_thread_inclusion fields

**Pull request merged**